modules/drivers/druid/test/metabase/driver/druid_test.clj

```diff
@@ -194,6 +194,12 @@
     {:aggregation [[:avg [:* $id $venue_price]]]
      :breakout    [$venue_price]}))
 
+;; share
+(expect-with-driver :druid
+  [[0.951]]
+  (druid-query-returning-rows
+    {:aggregation [[:share [:< $venue_price 4]]]}))
```

Comment:
Any reason not to port the other tests as well? Seems like more tests are always a good thing

Reply:
I would say this comment still stands, it can't really hurt to copy the other tests over to Druid too and if it's working correctly it shouldn't take more than 10 minutes or so

Reply:
Not all of them necessarily but more than just to make sure `:share-count` and nested stuff works ok on Druid